libcxx/include/__config

```diff
@@ -16,6 +16,17 @@
 #  pragma GCC system_header
 #endif
 
+#if defined(_LIBCPP_ENABLE_CXX17_REMOVED_FEATURES) && !defined(_LIBCPP_DISABLE_DEPRECATION_WARNINGS)
```

Comment:
note putting this in a different place in this header does not always generate the diagnostic. Since this is a short-living change I haven't investigated further.